### PR TITLE
genarate-module-component

### PR DESCRIPTION
**Pull request by dimahalaiskiy**

### CODE REVIEW

File: exadel-trainee-app/src/app/models/speaking/speaking.component.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-speaking',
+  templateUrl: './speaking.component.html',
+  styleUrls: ['./speaking.component.scss'],
+})
+export class SpeakingComponent implements OnInit {
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/speaking/speaking.component.ts:8]: You can see that @shuliakleonid in his PR named grammar test page (which is one of step of test as yours) as grammar-test, so its better to have common naming style.
So its better to change speaking to speaking test everywhere

**Comment by RomPavel** [comment]
You trying to add file, which is pretty huge and its not needed in repository

image

File: exadel-trainee-app/src/app/app.module.ts [hunk 2]
```diff
@@ -5,10 +5,11 @@ import { LoginModule } from './models/login/login.module';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { SpeakingModule } from './models/speaking/speaking.module';
 
 @NgModule({
   declarations: [AppComponent],
-  imports: [BrowserModule, LoginModule, AppRoutingModule, BrowserAnimationsModule],
+  imports: [BrowserModule, LoginModule, SpeakingModule, AppRoutingModule, BrowserAnimationsModule],
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/app.module.ts:12]: **SpeakingModule** should not be included into main module.

File: exadel-trainee-app/angular.json [hunk 3]
```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
```
Comment (Inunor) [inline exadel-trainee-app/angular.json:3]: What is it?)

File: exadel-trainee-app/src/app/models/speaking/speaking-routing.module.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { SpeakingComponent } from './speaking.component';
+
+const routes: Routes = [
+  {
+    path: 'speaking',
```
Comment (Inunor) [inline exadel-trainee-app/src/app/models/speaking/speaking-routing.module.ts:7]: should be just ''

**Review by Inunor — CHANGES_REQUESTED** [review]

**Pull request closed without merging**